distributed/diagnostics/tests/test_widgets.py

```diff
@@ -142,35 +140,20 @@ async def test_multi_progressbar_widget(c, s, a, b):
 
 
 @mock_widget()
-@gen_cluster()
-async def test_multi_progressbar_widget_after_close(s, a, b):
```

Comment:
This test is a bit odd and I couldn't make sense of the suffix `_after_close`. One thing that is unique about this test is that it tests the multiprogress bars explicitly with intermediate keys so I adjusted accordingly

Reply:
Keeps failing. After some thought I'm not even sure if this is possible. I'll remove the test